fix(curriculum): small english improvement (#42173)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/use-higher-order-functions-map-filter-or-reduce-to-solve-a-complex-problem.md

# --instructions--

> We have defined a function named `squareList`. You need to complete the code for the `squareList` function using any combination of `map()`, `filter()`, and `reduce()`. The `squareList` function should return a new array containing the squares of *only* the positive integers (decimal numbers are not integers) when an array of real numbers is passed to it. An example of an array of real numbers is `[-3, 4.8, 5, 3, -3.2]`.

Reviewer: ```suggestion
We have defined a function named `squareList`. You need to complete the code for the `squareList` function using any combination of `map()`, `filter()`, and `reduce()` so that it returns a new array containing the squares of *only* the positive integers (decimal numbers are not integers) when an array of real numbers is passed to it. An example of an array of real numbers is `[-3, 4.8, 5, 3, -3.2]`.
```
Reducing redundancy.

Author: I was under the impression that the original sentence was too long, making it harder to understand.
If there is someone else who agrees with @thecodingaviator 's wording, I'll happily accept.

Reviewer 2: I agree, the original sentence was too long.

Reviewer: How about
```suggestion
Complete the code for the `squareList` function using any combination of `map()`, `filter()`, and `reduce()` which has already been declared for you. The function should return a new array containing the squares of *only* the positive integers (decimal numbers are not integers) when an array of real numbers is passed to it. An example of an array of real numbers is `[-3, 4.8, 5, 3, -3.2]`.
```

Author: I like that but would also remove the last part making it:

"Complete the code for the `squareList` function using any combination of `map()`, `filter()`, and `reduce()`. The function should..."

I feel **which has already been declared for you** is redundant because **Complete** implies that the function is already declared, people see the function in the code editor to the right, and completing functions is nothing new at this point in the curriculum.

Let me know what you think

Reviewer: I've updated my suggestion above. I agree with you

Author: Nice, thank you.
